pages/about.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../styles/style.css" />
+    <link rel="stylesheet" href="../styles/about.css" />
+    <title>Microsoft</title>
+  </head>
+  <body>
+    <header class="desktop">
+      <div class="sub-header">
+        <div></div>
+        <ul>
+          <li>
+            <img class="social-icon" src="../images/home/facebook.svg" alt="" />
+          </li>
+          <li>
+            <img class="social-icon" src="../images/home/twitter.svg" alt="" />
+          </li>
+          <li>English</li>
+          <li>My page</li>
+          <li>Log out</li>
+        </ul>
+      </div>
+      <nav>
+        <div class="logo">
+          <img src="../images/microsoft-logo.svg" alt="" />
+        </div>
+        <ul>
+          <li><a class="abt" href="../index.html">Home</a></li>
+          <li><a class="abt" href="./about.html">About</a></li>
+          <li>Programe</li>
+          <li>Join</li>
+          <li>Sponser</li>
+          <li>News</li>
+          <li><a class="highlight" href="">Microsoft</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="hero">
+        <nav class="mobile">
+          <div class="burger">
+            <span></span>
+            <span></span>
+            <span></span>
+          </div>
+          <div class="mobile-menu">
+            <ul>
+              <li><a href="./index.html">Home</a></li>
```

Comment:
- [ ] THE link home doesn't work in about page it shows 404 not found please make sure to fix that so when you click on it takes you to home page